src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+
+  --link-color: #00acdc;
+  --header-heihgt: 60px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-self: center;
+  padding: 0 50px;
+}
+
+.logo {
+  padding: 10px 0;
+}
+
+.nav_list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  align-self: center;
+}
+
+.nav_link {
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  text-transform: uppercase;
+  position: relative;
+  display: block;
+  height: var(--header-heihgt);
+  line-height: var(--header-heihgt);
```

Comment:
We don't need 2 properties at once, choose and leave one of them.

Also, we don't need a variable for this, we will use this value only 1 time